### DISCUSSION TITLE
Add Whatsapp & Neat Download Manager.

### DISCUSSION THREAD

**Discussion by ghost** [chocolatey-community/chocolatey-packages#2469] [Ideas]
### Before starting, ensure that you have done the following.

- [X] I have verified that there is no other issue or discussion covering the feature or enhancement
- [X] I have verified there is no open pull request for the feature or enhancement
- [X] I have verified the package to improve is located in this repository

### What do you want to add?

Whats app.
Neat Download Manager.

### If the feature or enhancement is approved:

- [ ] I am willing to create a pull request to implement the feature or enhancement

**Comment by TheCakeIsNaOH** [comment]
Please open issues to request adding a new package at https://github.com/chocolatey-community/chocolatey-package-requests
